emulsify.info.yml

```diff
@@ -4,6 +4,9 @@ description: Theme using Storybook and component-driven development
 base theme: stable
 core: 8.x
 core_version_requirement: ^8 || ^9
+dependencies:
+  - components:components (>=8.x-2.x)
+  - emulsify_twig:emulsify_twig
```

Comment:
As a heads up... if you expect people to be pulling from github (as I suspect you do since your package is named `emulsify-ds/emulsify-drupal` vs `drupal/emulsify-drupal` see: https://github.com/emulsify-ds/emulsify-drupal/blob/2.x/composer.json#L2, then you are going to have to go the manual sync route for drupal dependencies. 

In that case... 

people can test PRs by doing something like `composer require emulsify-ds/emulsify-drupal:feat/gh-75-drupal-module-dependencies-dev --prefer-dist` but this may still create differences between dependencies if someone was to do `composer require drupal/emulsify_drupal` which is possible. This all may need to go to a different issue?

A thought, if another theme ever requires emulsify_drupal via their .info.yml file, they will get drupal/emulsify_drupal as a package added to their project automatically via drupal's packagist. There may be ways to counteract this.

Reply:
Want to add a final thing... in the composer installation steps https://docs.emulsify.info/installation/design-system#composer-recommended, you do mention `composer require emulsify-ds/emulsify-drupal`.